process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,180 @@
+
+
+#### "OpenHW Group Specification - Release Candidate"
+
+When the specification is proposed as a candidate for release, drafts will be entitled "OpenHW Group Specification: Title : VX.Y.Z  : Release Candidate"
+
+The Revision number used at this stage would normally be same as the proposed Revision to be released. 
+
+The Release Candidate specification will be notified as open for review to the OpenHW TWG.  
+Although technical comments are expected to have been resolved by the TG, comments may still be accepted at this stage.
+After a suitable review period, the TWG Chair/Co-Chair will determine that the Specification is ready for ballot. 
+
+
+#### "OpenHW Group Specification - Released"
+
+When a Release Candidate specification has completed ratification (through a TWG vote), the released copy will be entitled "OpenHW Group Specification: Title : VX.Y.Z  : Released"
+Revision numbers should be of the form X.Y.Z, normally starting with X=1 as the initially released version.
+
+The Github commit corresponding to the Released spec will be listed on the Github page of the specification repo together with the "OpenHW Group Specification: Title : VX.Y.Z : Released" string
+
+
+## Specification Format
+
+The following sections must be included in the OpenHW Group Specification
+
+### Title
+
+The Title should be of the form "OpenHW Group Specification: Title : VX.Y.Z : State"
+
+### Revision History
+
+Prior to release a table of draft revisions should be included, which can include description of content in each draft
+
+Upon release, the Revision History should list only released specification Revisions. That is, intermediate revisions used during development and review don't need to be listed.
+The table should include Revision, State, Date, and Description. The Description should include a high level description of the content.
+
+
+### License
+
+The Publication License is listed, together with a URL link to the source of the license. 
+
+
+The following is an example of an appropriate license statement:
+
+"
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+### No-warranty
+The following no-warranty text is included:
+
+"Unless required by applicable law or agreed to in writing, this Specification and any accompanying software or hardware distributed under the License are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License."
+
+### Copyright
+The following copyright text is included:
+
+"Copyright ©  YEAR OF PUBLICATION OpenHW Group. You may use, copy, modify, and distribute this work under the terms of the License, subject to the conditions specified in the License."
```

Comment:
the second sentence here should be part of the license text, not added to a copyright statement